tests/snippets/sysmod.py

```diff
@@ -6,6 +6,13 @@
 
 assert sys.platform == "linux" or sys.platform == "darwin" or sys.platform == "win32" or sys.platform == "unknown"
 
+try:
+    assert sys._framework == ""
```

Comment:
Maybe something that just checks that if there is the `_framework` attr, that it is a `str`?

```py
if hasattr(sys, "_framework"):
    assert type(sys._framework) is str
```